README.md

```diff
@@ -1,53 +1,51 @@
-# Role Name
+# mssql (WIP)
 ![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs and configures mssql on Red Hat Enterprise Linux 8.
+IMPORTANT: This role is WIP.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+* MS SQL requires a machine with at least 2000 megabytes of memory.
```

Comment:
FYI, This is a requirement that the `mssql-conf` returned me when I tried to run `mssql-conf setup` on a RHEL VM with less memory.

Reply:
ok - then we should add a check for that to tasks/main.yml - maybe even by using the `mssql-conf` command if it outputs a reasonable error message

Also, for CI testing, you'll need to specify that machines must have at least 2GB ram - see https://github.com/linux-system-roles/ha_cluster/pull/4#issuecomment-779910531 where we had to do something similar